### DISCUSSION TITLE
OpenCog live human data

### DISCUSSION THREAD

**Discussion by XxAvengerxX** [opencog/atomspace#3015] [General]
Is there way to use live human data and transfer to agi node plz someone help , ima not taking about photo and video etc ,live neural data from a person , I just need a place to start

**Comment by linas** [comment]
No. Four problems.

(1) Although you can record single neurons, or even hundreds of them with electrode arrays, or even hundreds of thousands with photoluminescent microscopy,you cannot do this at a whole brain level.  For example, you can't do it with MRI.

(2) AGI doesn't exist (yet).

(3) This is a graph database, not an AGI system.  This is the wrong place to talk about these things.

(4) If you were a star coder, I could show you how to process multiple parallel EEG streams.  It would be a fun project. But you'd have to be a coding wizard.

**Comment by XxAvengerxX** [comment]
Ok thank u

**Reply by XxAvengerxX** [reply]
Do u know where I can get something in this area ?

**Reply by linas** [reply]
Do you know any neuroscience? I mean, you're asking about stuff that top scientists spend their whole lives on. This is not something you do on a whim.  I mean, can you write code? Do you have access to EEG machines?

**Reply by XxAvengerxX** [reply]
Yes I been using my sons data for the last 4year

**Reply by XxAvengerxX** [reply]
Thank for ur time sir have a good day

**Reply by linas** [reply]
OK. Can you please hit the "accept answer" button, so that this doesn't stay open ass an issue forever?

Also, what is your "sons data"?  What is it ... some file containing ... what? MRI scans? EEG data? Megabytes? gigabytes? Where'd you get it? What format is it in?
